bindings/rust/s2n-tls/README.md

```diff
@@ -0,0 +1,2 @@
+This crate provides ergonomic, idiomatic Rust bindings for [s2n-tls](https://github.com/aws/s2n-tls). s2n stands for "signal to noise" and is a TLS implementation in C99.
```

Comment:
You probably don't need to explain what s2n means or repeat what the quote below says.